closeaccountingperiod.php

```diff
@@ -394,3 +394,60 @@ function closeaccountingperiod_civicrm_alterSettingsMetaData(&$settingsMetadata,
     'help_text' => '',
   );
 }
+
+/**
+ * Implementation of hook_civicrm_alterReportVar
+ */
+function closeaccountingperiod_civicrm_alterReportVar($varType, &$var, &$object) {
```

Comment:
Yes actually. I wanted to make sure that this part of code is executed only if the report is executed as an instance rather than the template since we need to have a contact ID in order for it to work. Since the template doesn't enforce this, it was not clear how I was going to add the filter for prior financial end month and year.